master/linking.html

```diff
@@ -161,14 +161,18 @@ <h3 id="URLandURI">URLs and URIs</h3>
 <h3 id="URLforms">Syntactic forms: URL and &lt;url&gt;</h3>
 
 <p>In SVG, most structural relationships between two elements
-are specified using a URL value in an <a href="#linkRefAttrs"><span
-class="attr-name">'href'</span></a> attribute.
-However, many <a>presentation attributes</a> allow both URLs and text strings as content.
-To disambiguate a text string from a relative URL, the <a>&lt;url&gt;</a>
-production is used for presentation attributes,
-and their corresponding CSS properties [<a href="refs.html#ref-css-values-3">css-values</a>].
-This is simply a URL delimited with a functional notation.
-</p>
+are specified using a <a>URL</a> value in an <a href="#linkRefAttrs"><span
+class="attr-name">'href'</span></a> attribute.</p>
+
+<p>This specification uses the terms <a>URL</a> and <a>&lt;url&gt;</a>.</p>
```

Comment:
Suggestion for the first two paragraphs, swapping the order & adding some clarification (note, I'm not sure if "apply suggestion" will work here; it only seems to work for one line at a time).

```suggestion
<p>To describe linking relationships,
this specification uses two different data types in attribute and property values:
<a>URL</a> and <a>&lt;url&gt;</a>.
The linking guidelines in this chapter apply to URLs specified with either syntax.
</p>

<p>The <a>URL</a> data type is a simple URL string.
In SVG, most structural relationships between two elements
are specified using a <a>URL</a> value in an <a href="#linkRefAttrs"><span
class="attr-name">'href'</span></a> attribute.
</p>
```

(and then continue with the paragraph about the `<url>` data type)

Reply:
@dirkschulze please accept (beta) the suggested change from @AmeliaBR and reply about the target of "must not"?